### PR TITLE
Synchronize

### PR DESCRIPTION
**Pull request by jrosser**
synchronize is based on rsync, which in turn requires ssh to the target. Things using a custom connection plugin (docker / lxc / lxd....) don't necessarily provide ssh to the target and synchronize fails.

### CODE REVIEW

**Comment by jrosser** [comment]
@mnaser is there anything else I need to do on this PR? We don't have SSH running in OSA containers so synchronize is pretty hard to use.

**Comment by jrosser** [comment]
@okozachenko1203 could you please take a look also - this should be low risk as the regular code path with the synchronize module is not changed

File: roles/upload_helm_chart/tasks/main.yml [hunk 1]
```diff
@@ -1,12 +1,7 @@
...
-  vexxhost.kubernetes.synchronize:
-    src: "{{ upload_helm_chart_src }}"
-    dest: "{{ upload_helm_chart_dest }}"
-    archive: false
-    recursive: true
-    checksum: true
-    delete: true
-    use_ssh_args: true
+- name: Upload helm char with synchronize
+  ansible.builtin.import_tasks: synchronize.yml
```
Comment (mnaser) [inline roles/upload_helm_chart/tasks/main.yml:2]: If we make this out to be `{{ upload_helm_chart_method }}.yml` would this save us from skipping a whole bunch of tasks?

**Review by mnaser — CHANGES_REQUESTED** [review]
small review

File: roles/upload_helm_chart/tasks/main.yml (hunk 1 above)
Comment (jrosser) [reply roles/upload_helm_chart/tasks/main.yml]: Done

**Pull request merged**